zerver/migrations/0177_user_message_add_and_index_is_private_flag.py

```diff
@@ -32,7 +32,7 @@ def reset_is_private_flag(apps: StateApps, schema_editor: DatabaseSchemaEditor)
             # `ORDER BY` clause that forces using the new index).
             flag_set_objects = (
                 UserMessage.objects.filter(user_profile__id=user_id)
-                .extra(where=["flags & 2048 != 0"])
+                .extra(where=["flags & 4096 != 0"])
```

Comment:
We can't readily renumber the `is_private` bit to a different slot; instead, we always add new bits at the end of the bitword.